--- kexec-tools-2.0.11/include/x86/x86-linux.h.orig	2015-09-02 03:03:16.000000000 +0200
+++ kexec-tools-2.0.11/include/x86/x86-linux.h	2016-01-18 21:53:37.749365567 +0100
@@ -21,9 +21,11 @@ struct e820entry {
 #define E820_RESERVED	2
 #define E820_ACPI	3 /* usable as RAM once ACPI tables have been read */
 #define E820_NVS	4
+} __attribute__((packed));
+#endif
+#ifndef E820_PMEM
 #define E820_PMEM       7
 #define E820_PRAM       12
-} __attribute__((packed));
 #endif
 
 /* FIXME expand on drive_info_)struct... */
--- kexec-tools-2.0.11/kexec/arch/i386/kexec-x86-common.c.orig	2015-10-06 10:53:46.000000000 +0200
+++ kexec-tools-2.0.11/kexec/arch/i386/kexec-x86-common.c	2016-01-18 21:53:12.732699950 +0100
@@ -43,6 +43,7 @@
 #ifdef HAVE_LIBXENCTRL
 #include <xenctrl.h>
 #endif /* HAVE_LIBXENCTRL */
+#include "x86/x86-linux.h"
 
 static struct memory_range memory_range[MAX_MEMORY_RANGES];
 
