--- kexec-tools-1.101/configure.ac.orig	2005-01-09 02:36:57.000000000 +0100
+++ kexec-tools-1.101/configure.ac	2005-08-28 12:12:20.729530896 +0200
@@ -93,8 +93,8 @@
 
 dnl ---Hard codes
 
-CFLAGS='-Wall -g -fno-strict-aliasing $(CPPFLAGS)'
-BUILD_CFLAGS='-O2 -Wall $(CPPFLAGS)'
+CFLAGS="$CFLAGS -Wall -fno-strict-aliasing \$(CPPFLAGS)"
+BUILD_CFLAGS="$CFLAGS -Wall \$(CPPFLAGS)"
 
 dnl ---Sanity checks
 if test "$CC"      = "no"; then AC_MSG_ERROR([cc not found]) fi
