--- kexec-tools-1.98/Makefile.main.orig	2004-09-15 09:49:17.000000000 +0000
+++ kexec-tools-1.98/Makefile.main	2004-12-14 22:25:32.325270400 +0000
@@ -2,7 +2,7 @@
 VERSION="1.98"
 DATE="15 September 2004"
 CPPFLAGS:= -I./util_lib/include -DVERSION='$(VERSION)' -DRELEASE_DATE='$(DATE)' $(EXTRA_FLAGS)
-CFLAGS:=-Wall -g -O2 -fno-strict-aliasing $(CPPFLAGS)
+CFLAGS:=-Wall -fno-strict-aliasing $(CPPFLAGS) $(OPTFLAGS)
 
 PREFIX:=$(OBJDIR)/build
 SBINDIR=$(PREFIX)/sbin
